conf/local.conf

```diff
@@ -81,3 +81,9 @@ UBOOT_SIGN_KEYNAME ?= "ubootdev"
 # custom signing key just generate a custom RSA 2048 key (PEM format) and
 # set via the OPTEE_TA_SIGN_KEY variable.
 OPTEE_TA_SIGN_KEY ??= "${TOPDIR}/conf/keys/opteedev.key"
+
+# Use Foundries to check and push an ostree repo to cloud storage
+USE_FIOTOOLS ?= "1"
```

Comment:
This will affect anyone building out of the public manifest, probably better to have it under the ci-scripts repo (and applied only for factory runs).

Reply:
That's my intention, we would like to get rid of garage-push and Treehub?
We already have `USE_FIOTOOLS` in ci-tools.

Reply:
Sure, then I think it is better to have this variable set at meta-lmp-base/classes/lmp.bbclass, as not every user is using this local.conf (or even this repo). There we can make meta-lmp users to use the right config.